src/Factories/FunFunFactory.php

```diff
@@ -1206,14 +1206,14 @@ private function getMembershipApplicationFormTemplate(): TwigTemplate {
 			'countries' => $this->getCountries(),
 			'addressValidationPatterns' => $validation->address,
 			'dateOfBirthValidationPattern' => $validation->dateOfBirth,
-			'incentives' => $this->getChoiceFactory()->getDefaultIncentives()
+			'incentives' => [ 'tote_bag' ]
```

Comment:
It's slightly dangerous to have the incentives as an array in 2 places, you could wrap them in a constant or method (and maybe even use it in the test). But that's just a nitpick, no blocker.